AnimView/frmMain.cpp

```diff
@@ -520,9 +531,10 @@ void frmMain::_onLastAnim(wxCommandEvent& evt) {
 }
 
 void frmMain::_onAnimChar(wxCommandEvent& evt) {
-  long iAnim;
-  if (evt.GetString().ToLong(&iAnim)) {
-    if (iAnim >= 0 && iAnim < (long)m_oAnims.getAnimationCount()) {
+  long conv;
+  if (evt.GetString().ToLong(&conv)) {
+    size_t iAnim = static_cast<size_t>(conv);
+    if (iAnim >= 0 && iAnim < m_oAnims.getAnimationCount()) {
       _onAnimChange((size_t)iAnim);
```

Comment:
iAnim is already a `size_t`

That also makes the  `>= 0` check one line higher useless.

Reply:
Yep. Sloppy. I have to range check then cast